Criipto.Signatures/Client.cs

```diff
@@ -0,0 +1,21 @@
+
+using GraphQL.Client.Abstractions;
+using GraphQL.Client.Http;
+using GraphQL.Client.Serializer.Newtonsoft;
+using System.Net.Http.Headers;
+
+namespace Criipto.Signatures {
+    public class CriiptoSignaturesClient : IDisposable {
+        private GraphQLHttpClient graphQLClient;
+
+        public CriiptoSignaturesClient(string clientId, string clientSecret, string endpoint = "https://signatures-api.criipto.com/v1/graphql") {
+            graphQLClient = new GraphQLHttpClient(endpoint, new NewtonsoftJsonSerializer());
+            graphQLClient.HttpClient.DefaultRequestHeaders.Authorization
+                = new AuthenticationHeaderValue("Basic", Convert.ToBase64String(System.Text.Encoding.ASCII.GetBytes($"{clientId}:{clientSecret}")));
+        }
+
+        public void Dispose() {
```

Comment:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1063

Reply:
@sgryt Hmm I'm not getting a warning for this rule.

Reply:
Nvm, had to add `<AnalysisMode>All</AnalysisMode>`